test/langtools/tools/javac/RestrictedMethods.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * @test /nodynamiccopyright/
+ * @bug 8316971
+ * @summary Smoke test for restricted method call warnings
+ * @compile/fail/ref=RestrictedMethods.out -Xlint:restricted -Werror -XDrawDiagnostics --enable-preview --source ${jdk.version} RestrictedMethods.java
```

Comment:
The `--enable-preview` flag related options will need to be removed when FFM is finalized